modules.cmake

```diff
@@ -43,25 +43,83 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif ()
 
 
+
+# Receives latest available C++ standard version
+#
+# Usage:
+#   modules_get_latest_cxx(<variable_name>)
+#   if(<variable_name> GREATER 17)
+#      ...
+#   endif()
+function(modules_get_latest_cxx result)
+  foreach(ver 26 23 20 17 14 11 98)
+    if(CMAKE_CXX_COMPILE_FEATURES MATCHES "cxx_std_${ver}")
+      set(${result} ${ver} PARENT_SCOPE)
+      break()
+    endif()
+  endforeach()
+endfunction()
+
+
+# Checks that compiler has support for C++ modules feature
+#
+# Usage:
+#   modules_is_supported(<variable_name>)
+#   if(<variable_name>)
+#      ...
+#   endif()
+function(modules_is_supported result)
```

Comment:
is looks a bit weird with plural modules. Maybe just `modules_supported`?

Reply:
renamed